src/main/scala/cromwell/engine/backend/jes/Run.scala

```diff
@@ -37,12 +39,29 @@ object Run  {
     def toRunMap = params.map(p => p.name -> p.gcs).toMap.asJava
   }
 
+  private def dateTryToString(dateTry: Try[DateTime]): String = dateTry match {
+    case scala.util.Success(d: DateTime) => d.toString
+    case Failure(e: Throwable) => e.getClass.getName
+  }
+
   sealed trait RunStatus
   trait TerminalRunStatus extends RunStatus
-  final case class Initializing(created: DateTime) extends RunStatus
-  final case class Running(created: DateTime, started: DateTime) extends RunStatus
-  final case class Success(created: DateTime, started: DateTime, finished: DateTime) extends TerminalRunStatus
-  final case class Failed(created: DateTime, started: DateTime, finished: DateTime, errorCode: Int, errorMessage: String) extends TerminalRunStatus
+  final case class Initializing(created: Try[DateTime]) extends RunStatus {
+    override def toString = s"Initializing(created: ${dateTryToString(created)})"
+  }
+
+  final case class Running(created: Try[DateTime], started: Try[DateTime]) extends RunStatus {
```

Comment:
Currently it tries to parse the date. If it fails or it's null, the exception type name is logged instead of the parsed datetime. No exceptions are thrown or workflows fail.